funcx_endpoint/funcx_endpoint/endpoint/interchange.py

```diff
@@ -320,6 +329,22 @@ def _main_loop(self):
             num_tasks_forwarded = 0
             num_results_forwarded = 0
 
+            if self._parent_pid:
+
+                def parent_watcher(ppid: int):
+                    print(f"WATCHING PARENT: {ppid}")
```

Comment:
This print statement is probably unintentional.

Reply:
Och, embarrassing.  Thanks, removed.